Fix howto in EGW docs (#422)
tigera/docs: calico-enterprise/networking/egress/egress-gateway-aws.mdx

- AssociateAddress
- DisassociateAddress

### Configure AWS Security Group rules

> In order to allow Egress Gateway traffic into the Egress Gateway pod's host from the client, the ingress rules of the security

Reviewer: ```suggestion
In order to allow egress gateway traffic into the egress gateway pod's host from the client, the ingress rules of the security
```

> group needs to be updated. A rule to allow all packets from within the security group must be added to the inbound rules.

Reviewer: ```suggestion
group need to be updated. A rule to allow all packets from within the security group must be added to the inbound rules.
```

- `egressIPSupport` must be the same on all cluster nodes, so you should set them only in the
  `default` FelixConfiguration resource.

> - As of v3.16.1, the operator will automatically enable the required policy sync API in the FelixConfiguration.

Reviewer: ```suggestion
- The operator automatically enables the required policy sync API in the FelixConfiguration.
```
Best not to talk about specific version numbers, particularly when we're talking about future capabilities. If this is landing in 3.16.1, we can consider leaving this out for now and adding that information along with the release.
